### DISCUSSION TITLE
zig的版本计划，有没有链接额，

### DISCUSSION THREAD

**Discussion by pyjcode** [zigcc/forum#105] [新手提问] [answered]
1. zig的版本计划
2. zig的发布周期

**Comment by jinzhongjia** [comment]
目前版本计划和发布周期是乱的，自从0.11被延迟后，core team那边有点强迫症，要感觉阶段任务完成的很不错后才会发布
外加这个周期目前和llvm发布周期（6个月）挂钩，所以一般会更长

**Reply by deatil** [reply]
zig不发布1.0版本感觉不怎么敢用，api改动太大了

**Reply by jinzhongjia** [reply]
> zig不发布1.0版本感觉不怎么敢用，api改动太大了

看你选择，我zig目前也作为半玩具语言，目前zig的工作是完善标准库和构建系统，构建系统的api已经趋于稳定了

**Reply by deatil** [reply]
只能先试下，有些标准库没定下来或者没有，没法正式用。web方向的库也没完善，想用下web都没法

**Reply by jinzhongjia** [reply, marked as answer]
如果只是为了web的话那py和go不是更好的选择吗

**Reply by pyjcode** [reply]
他可能是想通过比较熟悉的工作来学习zig，比如我，开始也是准备用写web来学习zig，现在我想写一些工具来学习他可能是想通过比较熟悉的工作来学习zig，比如我，开始也是准备用写web来学习zig，现在我想写一些工具来学习

**Reply by jinzhongjia** [reply]
很现实，zig的web生态很烂，对这方面感兴趣并且造轮子的很少
但是zig在玩具os,嵌入式，一些游戏，玩具语言，以及开发C abi插件上用的倒不少

**Reply by pyjcode** [reply]
不过，api不稳定确实是，我找到的很多开源项目想学学现在都跑步起来

**Reply by jinzhongjia** [reply]
给个小建议，可以以与 c 交互作为切入点来学习 zig, 我在以前觉得 c 是一个老掉牙的语言，但慢慢学的多了之后才明白，c 的这种老掉牙的标准也有很多好处（甚至于说可以理解为有些东西是刻意保留的），也理解为何C abi 会被大家默认为跨语言调用的标准
相对来说，一个语言抽象层次越高，就代表它为你底层遮蔽的东西越多，在我学习 zig 时，由于其的 low level 特性，我了解到不少在我以前用 go 用 ts 所不知道的东西
开源项目的 zig 项目，优先看近半年是否有更新，无更新的基本不用看了，大概率作者已弃坑。。。。

**Reply by deatil** [reply]
go这些的web写了好些了，想用下zig的。

**Comment by jinzhongjia** [comment]
这就需要手搓一些基础部件了，很繁琐，还得查很多资料

于 2024年4月2日 GMT+08:00 21:53:51, Deatil ***@***.***> 写到:
